src/modules/convert/GEN_STRUCT_DEMUX_fbt.cpp

```diff
@@ -106,23 +112,20 @@ bool GEN_STRUCT_DEMUX::createInterfaceSpec(const char *paConfigString, SFBInterf
   }
 
   // we could find the struct
-  auto structInstance = static_cast<CIEC_STRUCT *>(data.get());
+  var_IN = std::unique_ptr<CIEC_STRUCT>(static_cast<CIEC_STRUCT *>(data.release()));
```

Comment:
I would suggest directly assigning to `var_IN` above and removing `data`. Otherwise, please move assign the `unique_ptr` instead:
```suggestion
  var_IN = std::move(data);
```

Reply:
no I can't use std::move because I need to cast here. Doing this where data is created leads for non struct types to wrong results.

Reply:
Ah, I somehow overlooked the cast in there.